exporter/awsxrayexporter/internal/translator/sql.go

```diff
@@ -54,15 +56,23 @@ func makeSQL(attributes map[string]pcommon.Value) (map[string]pcommon.Value, *aw
 		return attributes, nil
 	}
 
-	if dbURL == "" {
-		dbURL = "localhost"
+	// Despite what the X-Ray documents say, having the DB connection string
+	// set as the URL value of the segment is not useful. So let's use the
+	// current span name instead
+	dbURL = span.Name()
```

Comment:
I'm hesitant about this change since it seems to break the schema for X-Ray segments. Also, how can we be sure the span name will always be a prettyfied query like the ones in the screenshot? Will other instrumentations name it differently?

Reply:
Hi @willarmiros,

Thanks for the feedback.
> I'm hesitant about this change since it seems to break the schema for X-Ray segments.

Yes... I do appreciate that this change goes against the schema for X-Ray. I am aware that allowing this change might force X-Ray to change their schema or at least the recommendations for values... However I do still think that the informational value of this change in the UI is good. But I can understand why it might be rejected.

> Also, how can we be sure the span name will always be a prettyfied query like the ones in the screenshot? Will other instrumentations name it differently?

This is a really good question. I don't think it's be possible to say how all instrumentations might name their DB spans. However I do think that OTel instrumentation authors should be following the official spec, [which does have guidance for span names](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span):
> The span name concisely identifies the work represented by the Span, for example, an RPC method name, a function name, or the name of a subtask or stage within a larger computation.